testing/trino-testing/src/main/java/io/trino/testing/BaseFailureRecoveryTest.java

```diff
@@ -209,7 +208,7 @@ public Object[][] parallelTests()
         };
     }
 
-    @Test(invocationCount = INVOCATION_COUNT, dataProvider = "parallelTests")
+    @Test(dataProvider = "parallelTests")
```

Comment:
looks good but maybe there was some TestNG weirdness, which explains why we had it.
@mwd410 @hashhar do you know maybe?

Reply:
FYI: It was changed from 3 in https://github.com/trinodb/trino/commit/d46c349ca4f75def1146fb65d913e03874a4feaa

Reply:
Yeah - thanks. Looks like remainder - not needed any more